src/map/chrif.cpp

```diff
@@ -1325,10 +1326,11 @@ int chrif_save_scdata(struct map_session_data *sd) { //parses the sc_data of the
 	WFIFOL(char_fd,8) = sd->status.char_id;
 
 	for (i = 0; i < SC_MAX; i++) {
```

Comment:
Iteration should also be improved!

Reply:
Agreed! This was something I had initially started to fix as well because we do this several times and it's expensive!

Reply:
We can do things like this, but we need this PR first